containers/java/Dockerfile

```diff
@@ -1,6 +1,7 @@
 # hadolint ignore=DL3006
 FROM alpine
 
+# hadolint ignore=DL3047
```

Comment:
this could be fixed  [with the suggestion from hadolint](https://github.com/hadolint/hadolint/wiki/DL3047). Is there a reason we don't want it?